src/App.vue

```diff
@@ -212,6 +174,24 @@ export default Vue.extend({
         this.$store.commit("setFetchError", null);
       }
     }
+  },
+  beforeCreate() {
+    // Configure the HTTP interceptors before anything else, to make sure HTTP
+    // requests behave as expected. (See the interceptors for details)
+    this.$store.dispatch("session/interceptRequests");
+
+    // Restore potential existing session from local storage.
+    const token = localStorage.getItem("jwt");
+    if (token !== null) {
+      this.$store.commit("session/login", JSON.parse(token));
+    }
+
+    // Regularly refresh the authorization token.
+    this.$store.dispatch("session/refreshTokenLoop");
+
+    // Now fetch the user data, as session/token logic is ready and will ensure
```

Comment:
Love the comments!  I'm making a mental note to do that everywhere too from now on.